spin/spin.go

```diff
@@ -55,8 +62,11 @@ func commandStart(command []string) tea.Cmd {
 		}
 		cmd := exec.Command(command[0], args...) //nolint:gosec
 
-		cmd.Stdout = &outbuf
-		cmd.Stderr = &errbuf
+		stdout := io.MultiWriter(&bothbuf, &errbuf)
+		stderr := io.MultiWriter(&bothbuf, &outbuf)
```

Comment:
By doing this the `bothbuf` will have the output of both STDERR and STDOUT but it will be in the order it was written by the process. 

If we have the buffers entirely separate and the writes to each were interleaved in any way, this might cause the output to be confusing or be inaccurate.